FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/MessagesListFragment.kt

```diff
@@ -1431,6 +1425,28 @@ class MessagesListFragment : BaseFragment<FragmentMessagesListBinding>(), ListPr
     }
   }
 
+  private fun handleOutboxMessagesChanges(it: List<MessageEntity>) {
+    val msgsCount = it.size
+    supportActionBar?.subtitle = if (it.isNotEmpty() && currentFolder?.isOutbox == false) {
+      when {
+        it.any { messageEntity ->
+          messageEntity.msgState == MessageState.SENDING
+        } -> {
+          getString(R.string.sending_message)
+        }
+
+        else -> resources.getQuantityString(R.plurals.outbox_msgs_count, msgsCount, msgsCount)
+      }
+    } else null
+
+    isForceSendingEnabled = msgsCount > 0
+    isForceSendingEnabled = it.none { entity -> entity.msgState == MessageState.SENDING }
```

Comment:
looks like `isForceSendingEnabled = msgsCount > 0` line isn't needed, as `isForceSendingEnabled` is re-assigned on the next line?

Reply:
Good one. Thank you!
It's old code. Interesting, when I added this "great" code :)
I will fix it soon.